[VIT-3701] Add docs for registrable testkit flow (#122)
tryVital/docs: docs/lab/workflow/order-unregistered-testkit.mdx

---

> title: "Ordering an Unregistered Testkit"

Reviewer: ```suggestion
title: "Ordering an Unregistered Test Kit"
```

The product name is [At-home Test Kits](https://tryvital.io/labs)

---

> Vital provides a way to order testkits for other patients (registrable testkits). 
> 
> Under the [normal order flow](/api-reference/lab-testing/create-order), you place an order for a specific patient. This order is then always associated with that patient and cannot be passed on to someone else.
> 
> In order to allow for `testkits` to be purchased and passed on to another patient, Vital provides the `registrable testkit` flow.

Reviewer: ```suggestion
Under the [standard Testkit ordering flow](/api-reference/lab-testing/create-order), you place a Testkit order for a specific patient. This resulting Testkit is bound to that patient, and the kit cannot be passed on to someone else.

With Unregistered Testkits, Vital offers a different option — Testkits not bound to any specific patient can be ordered to a specific household address. The patient registration process is deferred until an actual patient intends to use the Testkit.
```

<Warning>

>   Note that this only applies to `testkits`, not `walk-in phlebotomy` or `at-home phlebotomy`.

Reviewer: These are references to the product feature, not to a code entity.

Author: Not sure what your point is here

Author: AH I see

</Warning>

> This involves two steps from the customer's perspective:

Reviewer: who is "the customer"?

  1. Ordering the kit.
  2. Registering a patient.

After step 1, the order is sent to the requested address and generates all the same webhooks as a regular order would. However it is stuck on the `awaiting_registration` state, meaning that no requisition form is generated for this order, and no results can be obtained until step 2 is done.

> After step 2, the order flow resumes as normal, the patient sends the `testkit` to the lab, the order is progressed to the `testkit_registered` state and, again, all the same webhooks as in the regular order flow are dispatched.

Reviewer: Full status name? Isn't the status is three part with dot separator?

Also /state/status/g

    "street": "123 Main St.",
    "street_number": "Apt. 208",
    "city": "San Francisco",
    "state": "CA",
    "zip": "91189",
    "country": "US",
    "phone_number": "+11234567890"
  }
)
```

The `testkit` is now ordered, and will be sent to the specified address. Once delivered, it can be kept until it's expiration. 

Once the `testkit` is with it's intended patient, it should be registered (step 2).

In order to register, Vital requires a `sample_id`, which is found within the `testkit` itself. This is an unique identifier. 
Vital also requires the patient details and address.
Besides this, if using Vital's physician network, then supplying the consents field is required. If providing your own physician, then the physician information is required.

For this example, we will assume the use of your own physician.

To register a `testkit` order, make a request to our [`POST /v3/order/testkit/register/`](/api-reference/lab-testing/register-order), to fulfil step 2.

```python Python
client = Client(api_key, "sandbox")

data = client.LabTests.register_testkit_order(
  user_id="63661a2b-2bb3-4125-bb1a-b590f64f057f",
  sample_id="123123123",
  patient_details={

>       "first_name": "John",
>       "last_name": "Doe",
>       "dob": "2020-01-01",
>       "gender": "male",
>       "phone_number": "+1123456789",
>       "email": "email@email.com"
>   },
>   patient_address={
>     "receiver_name": "john Doe",
>     "street": "123 Main St.",
>     "street_number": "Apt. 208",
>     "city": "San Francisco",
>     "state": "CA",
>     "zip": "91189",
>     "country": "US"
>   },

Author: ```suggestion
    "first_name": "John",
    "last_name": "Doe",
    "dob": "2020-01-01",
    "gender": "male",
    "phone_number": "+1123456789",
    "email": "email@email.com"
  },
  patient_address={
    "receiver_name": "john Doe",
    "street": "123 Main St.",
    "street_number": "Apt. 208",
    "city": "San Francisco",
    "state": "CA",
    "zip": "91189",
    "country": "US"
  },
```
